spec/system/batch_edit_spec.rb

```diff
@@ -33,6 +33,45 @@
   }
 
   it "smoke test" do
+
+    # Admin works page
+
+    visit admin_works_path
+
+
+    # Check all items
+    find("#check-or-uncheck-all-works").check
+
+    # Remove this line at your own risk.
+    stall =  [work0, work1, work2, work3,].map {|w| find("#cartToggle-#{w.friendlier_id}").checked?}
```

Comment:
Workaround for flaky behavior; this will have to do for now.

Reply:
this actually seems reasonable i guess